Gemfile

```diff
@@ -11,8 +11,6 @@ group :maintenance, optional: true do
   gem "rubocop-packaging", require: false
 end
 
-gem "psych", "~> 4.0"
```

Comment:
Do you mind explaining why we should remove this? Is it breaking something? Current CI seems all green to me.

Reply:
IMO, it is unreasonable to expect people to upgrade stdlib gems.  Tests should pass on any supported Ruby installation with the stdlib gems of that Ruby version (specifying non stdlib gems, such as webrick in Ruby 3+, is fine).  To make sure that tests still pass with the stdlib gems, it should not be specified in the Gemfile used for CI.  The tests were broken in Ruby 2.4-2.7 with stdlib.

Reply:
Fair enough. I don't fully agree with this, but it's fine. The problem here is the stdlib interface was basically broken in a way that requires a monkey patch for compatibility if we need to support old versions.